packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -36,18 +52,28 @@ import {createUrlTemplate} from '../../utils/url'
 const AppConfig = ({children, locals = {}}) => {
     const [basket, setBasket] = useState(null)
     const [customer, setCustomer] = useState(null)
-
     return (
         <MultiSiteProvider site={locals.site} locale={locals.locale} buildUrl={locals.buildUrl}>
-            <CommerceAPIProvider value={locals.api}>
-                <CustomerProvider value={{customer, setCustomer}}>
-                    <BasketProvider value={{basket, setBasket}}>
-                        <CustomerProductListsProvider>
-                            <ChakraProvider theme={theme}>{children}</ChakraProvider>
-                        </CustomerProductListsProvider>
-                    </BasketProvider>
-                </CustomerProvider>
-            </CommerceAPIProvider>
+            <_CommerceAPIProvider value={locals.api}>
```

Comment:
Need to keep the old Provider until we can completely remove old commerce-api code to avoid any breaking in the app